Source/SwiftLintFramework/Rules/CustomRules.swift

```diff
@@ -50,11 +50,25 @@ public struct CustomRules: Rule, ConfigurationProviderRule {
     public init() {}
 
     public func validateFile(file: File) -> [StyleViolation] {
-        if configuration.customRuleConfigurations.isEmpty {
+        var configurations = configuration.customRuleConfigurations
+
+        if configurations.isEmpty {
             return []
         }
 
-        return configuration.customRuleConfigurations.flatMap {
+        if let path = file.path {
+            configurations = configurations.filter {
```

Comment:
could you please use a named block parameter here instead of `$0`? Maybe `{ configuration in`